### PR TITLE
feat: discover pages screen mp 598

### PR DESCRIPTION
**Pull request by christian14b**
Needed to change the structure of previous work to make the job of adding more "pages" simpler. 

Desktop

https://github.com/user-attachments/assets/bcd251a4-b94f-4b72-a271-6421b4a94c0c

Mobile

https://github.com/user-attachments/assets/50ec9d53-b5cd-41ee-a661-eb4293c504fd

### CODE REVIEW

**Comment by roger-in-kiva** [comment]
Looks like `Discover more badges` button is not full width.

Screenshot 2024-08-16 at 1 14 03 p m

**Comment by dyersituations** [comment]
@christian14b does this page work well for desktop as well? If so, please post a screenshot/video for that use case.

**Comment by dyersituations** [comment]
> > @christian14b does this page work well for desktop as well? If so, please post a screenshot/video for that use case.
> 
> Yes, the video is in desktop! I'll attach a mobile view

Oh really? The video looks like the mobile emulator.

**Comment by dyersituations** [comment]
@christian14b I see the link to the desktop video now. It appears to be in a box, is that expected now? The mocks show the desktop as the full screen under the nav:
image

File: src/components/Thanks/BadgesCustomization.vue [hunk 1]
```diff
@@ -1,274 +1,299 @@
...
-					</kv-expandable>
 				</div>
 			</div>
 		</div>
+		<transition name="fade">
+			<DiscoverSection
+				v-if="discoverBadges"
+				:selected-loan-region="selectedLoanRegion"
+				:is-guest="isGuest"
+				@back-to-earned-badge="discoverBadges = false"
```
Comment (dyersituations) [inline src/components/Thanks/BadgesCustomization.vue:294]: `() => discoverBadges = false` to help with later migration to Vue 3.

File: src/components/Thanks/BadgesCustomization.vue [hunk 2]
```diff
@@ -593,4 +551,12 @@ export default {
 	@apply tw-w-full tw-h-full tw-top-0 tw-left-0 tw-transform-none;
 }
 
+.fade-enter-active {
+	transition: opacity 1s;
+}
+
+.fade-enter {
+	opacity: 0;
```
Comment (dyersituations) [inline src/components/Thanks/BadgesCustomization.vue:559]: `@apply tw-opacity-0`

File: src/components/WwwFrame/WwwPage.vue [hunk 3]
```diff
@@ -84,15 +88,31 @@ export default {
...
+		if (tyBadgeExpData?.version) {
+			this.enableTYBadgesExp = tyBadgeExpData?.version === 'b';
+		}
 	},
 	computed: {
 		mainClasses() {
 			return [
 				this.mainClass,
 				{ 'tw-bg-secondary': this.grayBackground },
+				{ 'tw-relative': this.badgesExperimentEnabled }
```
Comment (dyersituations) [inline src/components/WwwFrame/WwwPage.vue:106]: Can this be applied to the `WwwPage` from the thanks page?

Comment (dyersituations) [reply src/components/WwwFrame/WwwPage.vue:106]: Sounds good. I was just thinking there might be a way to consolidate the experiment logic to the thanks page.

**Pull request merged**